MdeModulePkg/Bus/Pci/XhciDxe/Xhci.c

```diff
@@ -458,7 +458,7 @@ XhcGetRootHubPortStatus (
   // For those devices behind hub, we get its attach/detach event by hooking Get_Port_Status request at control transfer for those hub.
   //
   ParentRouteChart.Dword = 0;
-  XhcPollPortStatusChange (Xhc, ParentRouteChart, PortNumber, PortStatus);
+  Status                 = XhcPollPortStatusChange (Xhc, ParentRouteChart, PortNumber, PortStatus); // MU_CHANGE

```

Comment:
Why so much whitespace?

Reply:
This was because the uncrustify check was failing if it was unaligned with the `=` on line 460: 
```
Status = XhcPollPortStatusChange (Xhc, ParentRouteChart, PortNumber, PortStatus); // MU_CHANGE
```

Reply:
Ah, of course. It was [visually] masked by the red "removed" line.